daemon/libnetwork/internal/resolvconf/resolvconf.go

```diff
@@ -71,7 +69,7 @@ type ExtDNSEntry struct {
 
 func (ed ExtDNSEntry) String() string {
 	if ed.HostLoopback {
-		return fmt.Sprintf("host(%s)", ed.Addr)
+		return "host(" + ed.Addr.String() + ")"
```

Comment:
Updated this one in the last iteration as well to shave off some bits;

Before/after;

```
    BenchmarkGenerate-10     2617477       453 ns/op      736 B/op      8 allocs/op
    BenchmarkGenerate-10     3148328       373 ns/op      712 B/op      7 allocs/op
```